ktor-client/ktor-client-android/jvm/src/io/ktor/client/engine/android/Android.kt

```diff
@@ -25,6 +25,7 @@ import io.ktor.client.engine.*
  * ```
  *
  * You can learn more about client engines from [Engines](https://ktor.io/docs/http-client-engines.html).
+ * [Report a problem](https://ktor.io/feedback?fqname=io.ktor.client.engine.android.Android)
```

Comment:
Should we add a blank line between "Report a problem" and a comment body? To make it displayed on a separate line.

Reply:
sure, let me fix the generator

Reply:
@osipxd, please check.

Reply:
The generator can also be used on CI checks and it takes about 2 seconds to run on the whole code base